bin/node-template/pallets/template/Cargo.toml

```diff
@@ -20,6 +20,7 @@ scale-info = { version = "2.1.1", default-features = false, features = ["derive"
 frame-benchmarking = { version = "4.0.0-dev", default-features = false, optional = true, path = "../../../../frame/benchmarking" }
 frame-support = { version = "4.0.0-dev", default-features = false, path = "../../../../frame/support" }
 frame-system = { version = "4.0.0-dev", default-features = false, path = "../../../../frame/system" }
+sp-std = { version = "5.0.0", default-features = false, path = "../../../../primitives/std" }
```

Comment:
You just round yourself another issue (if you want obviously:stuck_out_tongue:).  

Getting rid of this and using `PhantomData` from `core::marker` instead.